automation/src/test/scala/org/broadinstitute/dsde/firecloud/auth/ServiceAccountAuthToken.scala

```diff
@@ -0,0 +1,32 @@
+package org.broadinstitute.dsde.firecloud.auth
+
+import java.io.ByteArrayInputStream
+
+import akka.actor.ActorSystem
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import org.broadinstitute.dsde.firecloud.config.Config
+import org.broadinstitute.dsde.workbench.google.HttpGoogleIamDAO
+import org.broadinstitute.dsde.workbench.google.model.GoogleProject
+import org.broadinstitute.dsde.workbench.model.WorkbenchUserServiceAccountEmail
+
+import scala.collection.JavaConverters._
+import scala.concurrent.{Await, ExecutionContext, ExecutionContextExecutor}
+import scala.concurrent.duration._
+
+// Note: we are creating a new auth token from google every time we call this case class
```

Comment:
Maybe could say "a new service account private key" to be more precise

Reply:
I think I wrote that comment originally so that's my fault. It could be that I got this wrong. I'd like to understand this better so I can use the right terminology going forward.